### DISCUSSION TITLE
Challenge proof time limit

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#2050] [Q&A]
Currently, I build up a set of disks array. I test it with 3 plots and using `./chia plots check -n 10` to test the plots.
The runtime and the proofs for these 3 plots are:
8min15sec - 14/10
7min41sec - 12/10
7min13sec - 14/10
All of these plots are k32.
The average time of finding a single proof is (8m15s+7m41s+7m13s)/(14+12+14)=34.725s
My random access of disk is really kind of slow, but it is parallelizable. Are there any ways to use the ability of parallelizing access to speed up the proof check?
Is my test result capable of farming? It is hard to improve that since my drive array structure limitation.
Thank you!

**Comment by hoffmang9** [comment]
Chia plots check behaves like you have a winning proof for every attempt. Actual farming works very differently and is already multi-threaded.

**Reply by cccat6** [reply]
> Chia plots check behaves like you have a winning proof for every attempt. Actual farming works very differently and is already multi-threaded.

Thank you.
I wondering if the performance of proofing speed is enough for actual farming since is it significantly slower than a regular hard drive.
The parallelizing access that I mentioned is not about multi-thread for CPU, but for requests for plot files on drives.
